bftengine/src/preprocessor/PreProcessor.cpp

```diff
@@ -1110,6 +1110,17 @@ void PreProcessor::cancelPreProcessing(NodeIdType clientId, uint16_t reqOffsetIn
   }
 }
 
+PreProcessResultSignature signPrimaryPreExecutionResultHash(const ReplicaId &myId,
+                                                            const concord::util::SHA3_256::Digest &hash) {
+  auto sm = SigManager::instance();
```

Comment:
Should we add a check that this function will get executed if myId is of primary in the given view. If not then just return, so that any accidental call by other replica can be avoided.

Reply:
After addressing Yulia's comments this code is now gone.